spec/models/withdrawal_reason_spec.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require "rails_helper"
```

Comment:
Does something need to go in here?

Reply:
hmm possibly crept back after a rebase - have removed now